airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -201,7 +215,7 @@ def _check_success_task(self):
             # This is a `stoppedReason` that indicates a task has not
             # successfully finished, but there is no other indication of failure
             # in the response.
-            # See, https://docs.aws.amazon.com/AmazonECS/latest/developerguide/stopped-task-errors.html # noqa E501
+            # See, https://docs.aws.amazon.com/AmazonECS/latest/developerguide/stopped-task-errors.html  # noqa: E501  # pylint: disable=line-too-long
```

Comment:
Why we need to have "pylint-line-too-ling"?
https://github.com/apache/airflow/blob/master/pylintrc#L346-L347

Reply:
Turns out it's because `See, ` is not captured by the regex:
```
^\s*(# )?<?https?://\S+>?$
```

Only long lines where https://.... is directly after the # are ignored.

I could change the regex to include any characters after # and before https://:

```
^\s*(# )?.*<?https?://\S+>?$
```

Or just leave this case as it is right now, WDYT?

Reply:
Maybe add enter just? Then the link will be in a new line.

Reply:
fixed